prebuilt-config/omnibus/bitops.config.yaml

```diff
@@ -13,6 +13,18 @@ bitops:
     filename: bitops-run      # log filename
     err: bitops.logs          # error logs filename
     path: /var/logs/bitops    # path to log folder
+    # Define the secrets to mask
+    masks:
+      - # regex to search
+        # looks for `BITOPS_KUBECONFIG_BASE64={string}`
+        search:  (.*BITOPS_KUBECONFIG_BASE64.*\=)(.*\n)
```

Comment:
If we're targeting beginners, I wouldn't expose regex setting in the BitOps config as it brings complexity to the user's side. It's enough to list 10 regex rules to make everything look complicated and repell someone using it.

Other options:
1) Just list the secret vars in bitops core or config. No regex, just BITOPS_KUBECONFIG_BASE64, AWS_SECRET_KEY, etc. and do the rest filtering on the BitOps side.
2) auto-detect secrets in log via heuristics and rules from libs like https://github.com/Yelp/detect-secrets/tree/master/detect_secrets/plugins
3) allow specifying extra secret names on top of default/hardcoded

```yaml
secrets:
  - BITOPS_KUBECONFIG_BASE64
  - AWS_SECRET_ACCESS_KEY
  - AWS_ACCESS_KEY_ID
  - AWS_SESSION_TOKEN
```

Alternatively, allow defining secrets on BitOps Plugin level and process in the code so the secret definition is pluggable.